frontend/public/components/machine-autoscaler.tsx

```diff
@@ -41,11 +41,11 @@ const MachineAutoscalerTargetLink: React.FC<MachineAutoscalerTargetLinkProps> =
 };
 
 const tableColumnClasses = [
-  'pf-u-w-33-on-lg',
   '',
-  'pf-m-hidden pf-m-visible-on-md',
-  'pf-m-hidden pf-m-visible-on-lg pf-u-w-16-on-lg',
-  'pf-m-hidden pf-m-visible-on-lg pf-u-w-16-on-lg',
+  '',
+  'pf-u-w-25-on-xl pf-m-hidden pf-m-visible-on-sm',
+  'pf-u-w-10-on-2xl pf-u-w-16-on-xl pf-m-hidden  pf-m-visible-on-lg',
```

Comment:
It seems like in the other instances you are doing : `hidden visible size`.  Would be good to be consistent.

Reply:
Nit:  extra space between `pf-m-hidden` and `pf-m-visible-on-lg`